k2/csrc/ragged_ops.h

```diff
@@ -197,36 +197,42 @@ RaggedShape Stack(int32_t axis, int32_t src_size, RaggedShape **src,
     @param [out] out  The container where the output RaggedShapes would write
```

Comment:
Around line 192, I think we have to have a clearer documentation of the purpose of this function.
Is it the case that we are trying to do the opposite of Stack(), i.e. to generate an array `out` such that
`Equal(src, Stack(axis, out->size(), out->data()))` ?